### ISSUE TITLE
espresso package command gives back incorrect command options

### ISSUE THREAD

**Issue by mway-io** [mway-io/Espresso#62]
> espresso package 

Usage: espresso package [<options>]
Available options:
   -d, --directory=DIRECTORY   Specify a custom project directory Default: $PWD
   -h, --help                  Show this help for command package Default: false

From my point of view that's wrong when I'm comparing with the documentation here:

espresso package <mode> <configName>

**Comment by josher19** [comment]
Should give the same output as "espresso help package":

```
Usage: espresso package <mode> [<target>]

Available modes:
   run                         Build and run project in emulator.
   build                       Build native application package.

Available targets:
   any property-key of package in your config.json; default: "default"
```

and also provide a link to your wiki which discusses the "package" section of config.json

**Comment by 4z3** [comment]
This is fixed in 6862094cc172af69bb29a52a7b30b84c5d876aab
